teachopencadd/talktorials/T013_query_pubchem/talktorial.ipynb

```diff
@@ -6,10 +6,13 @@
    "source": [
```

Comment:
<blockquote>Use <strong>a</strong> <code><s>SMILES</s></code><s> </s><strong>SMILES </strong>string to <s>query</s> <strong>search for</strong> similar compounds in PubChem.</blockquote><blockquote><s>In the following </s><strong>Using the</strong> function, we will query the PubChem service for similar compounds to the given one. The fingerprint Tanimoto-based 2-dimensional similarity will be calculated here. You can see <a href="https://jcheminf.biomedcentral.com/articles/10.1186/s13321-016-0163-1" target="_blank">this paper</a> for more details about similarity evaluation in PubChem, and you can also see <a href="https://ftp.ncbi.nlm.nih.gov/pubchem/specifications/pubchem_fingerprints.pdf" target="_blank">this specification</a> for the details about PubChem fingerprint.</blockquote><p> </p><blockquote>The fingerprint Tanimoto-based 2-dimensional similarity</blockquote><p>Maybe "The Tanimoto-based similarity between (2D) XXX fingerprints will be calculated", where XXX defines the type of fingerprint they use? XXX = PubChem substructure?</p><p> </p><blockquote>This job will be done <s>by </s><strong>in</strong> four main steps:</blockquote><blockquote>Input the canonical <s>smile</s> <strong>SMILES </strong>string for the given compound to create a new task, so we obtain a job key here. Note that this asynchronous API will not return the data immediately. In this case, the callback will be provided only when the requested resource is ready, which <strong>can be </strong>checked by the job key. So asynchronous requests are useful to work around certain slower operations.</blockquote><p> </p><blockquote>So, be nice and wait for the callback.</blockquote><p>Unsure about the wording here but I don't know.  I didn't understand in what way we are supposed to be nice. Maybe something like "Make sure you wait a bit between your requests to be nice to the server"?</p><p> </p><blockquote>Get the canonical <s>smiles </s><strong>SMILES string</strong> from the PubChem CIDs.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>done</p>